### ISSUE TITLE
Root cause of Touch and Performance issues

### ISSUE THREAD

**Issue by dratwas** [skrafft/react-native-jitsi-meet#343]
Hey all, recently I found this library and was about to use it, however, I decided not for some reason. I do not write that you should not use this library. I don't even have an alternative, but you should know how it works and how it affects your applications.

So this library is only a binding for the "native" Jitsi-meet SDK which is here [iOS](https://github.com/jitsi/jitsi-meet/tree/master/ios) and [Android](https://github.com/jitsi/jitsi-meet/tree/master/android). This (react-native-jitsi-meet) library does a really good job, however, the Jitsi SDK is not a native SDK :D Maybe some part is native but... the fun fact is that it runs React-Native inside.
iOS - (https://github.com/jitsi/jitsi-meet/blob/master/ios/sdk/src/JitsiMeetView.m#L200-L212)
Android - (https://github.com/jitsi/jitsi-meet/blob/master/android/sdk/src/main/java/org/jitsi/meet/sdk/BaseReactView.java#L132-L148)

This is the reason why touch events don't work and performance is suboptimal (memory usage, CPU usage etc). Touch events are handled by the wrong instance of react-native. Your application size also will grow since jitsi-sdk framework includes all dependencies in it, so your applications will contain react-native two times.

There are probably more downsides in running react-native in react-native. In the future, there could be even more issues.

TL;DR Running the official jitsi-meet SDK in react-native application is not a good idea since the jitsi-meet SDK runs react-native inside and you will end with two react-native instances, in most cases with a different version.

@skrafft As I mentioned, you did a great job implementing this library and I can not say a bad word about this but I have one question. WDYT about adding a note about that in the readme? I think it is important that people, that want to use this library, know that it runs a new react-native instance inside the JitsiView.

**Comment by skrafft** [comment]
Hi @dratwas,

This library is indeed React Native in React Native. Sadly, there are sadly no other choices if you want to use JitsiMeet SDK.
We've been using this solution in production at ibani and Mt Pelerin for 3 years now with no performance or touch issues and have processed around 2000 compliance video calls with customers.

That being said, I agree with you that we could improve the README to mention this. I'll do it.

**Comment by skrafft** [comment]
@dratwas done, feel free to submit a PR with further changes to the readme if you think I missed some key items

**Comment by riyas145** [comment]
then what is the solutions ? @skrafft  @dratwas
